services/src/unit-test/java/uk/gov/moj/sdt/services/BulkFeedbackServiceTest.java

```diff
@@ -126,23 +124,20 @@ public class BulkFeedbackServiceTest extends AbstractSdtUnitTestBase {
     /**
      * Setup of the mock dao and injection of other objects.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
This should be able to be removed as the ExtendWith(MockitoExtension.class) annotation initialises the annotated mocks in the class.

Reply:
removed

Reply:
Confirmed removal of openMocks() line.